lib/widgets/twake_components/twake_preview_placeholder.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+class TwakePreviewPlaceHolder extends StatelessWidget {

```

Comment:
is this necessary ? , why don't you use directly with `CircularProgressIndicator`

Reply:
yes, I want it to have the same size as the previewed link and I want to center the indicator otherwise it's flatten